### DISCUSSION TITLE
Is there any reason to use ARP scan and db import together?

### DISCUSSION THREAD

**Discussion by bwirt** [netalertx/NetAlertX#456] [Q&A]
I'd like to put Pi.Alert behind my Traefik proxy, but can't find a way to do this easily while the container is running in network host mode.  Since I'm using the Unifi db import scan, is there any point at all to leaving the arp scan enabled as well?  If I turn this off along with network host mode, I should be able to move pi.alert to my proxy network.

**Comment by jokob-sk** [comment]
Hey there,

I don't think so. Any one of the plugins writing into the CurrentScan DB table and thus responsible for discovering devices is sufficient.

More info on what plugins write into that table here:

https://github.com/jokob-sk/Pi.Alert/blob/main/front/plugins/README.md

Also, if you set up Traefik, can you please help with answering this discussion?

https://github.com/jokob-sk/Pi.Alert/discussions/449

Thanks in advance! 
J

**Comment by bwirt** [comment]
Thanks, the problem was my mistake, I was able to get Traefik working even without disabling the ARP scan.  Pi.alert is working nicely with https now.

I took a look at #449 but I'm afraid I don't have any insight here as I'm new to Traefik and am only using the hostname directly (traefik.mydomain.com) instead of a path (my.domain.com/pialert).

**Reply by jokob-sk** [reply]
Thanks for the response! I linked this answer in the other discussion for others to find it if needed.
